app/Actions/Server/ConfigureTrafficAnalytics.php

```
@@ -0,0 +1,27 @@
+class ConfigureTrafficAnalytics
+{
+    use AsAction;
+
+    public function handle(Server $server, bool $enable): void
+    {
+        $server->settings->is_traffic_analytics_enabled = $enable;
+        $server->settings->save();
+        $server->refresh();
+
+        // Regenerate proxy config so the (Traefik) access-log flags / (Caddy) log labels take effect.
+        GetProxyConfiguration::run($server, forceRegenerate: true);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Actions\Proxy\GetProxyConfiguration' in method 'handle'. (undefined)

(StaticAccess)